packages/wdio-logger/src/node.js

```diff
@@ -128,12 +128,16 @@ export default function getLogger (name) {
 getLogger.setLevel = (name, level) => loggers[name].setLevel(level)
 getLogger.setLogLevelsConfig = (logLevels = {}) => {
     logLevelsConfig = {}
+
     Object.keys(logLevels).forEach(loggerName => {
         const logLevelName = getLogLevelName(loggerName)
-
         logLevelsConfig[logLevelName] = logLevels[loggerName]
+    })
+
+    Object.keys(loggers).forEach(loggerName => {
```

Comment:
it might be clearer to name it `logger` instead of `loggerName`

Reply:
will do within next PR